app/src/sharedTest/java/org/oppia/android/app/recyclerview/RecyclerViewMatcher.kt

```diff
@@ -74,6 +85,114 @@ class RecyclerViewMatcher {
     fun hasGridColumnCount(expectedColumnCount: Int): ViewAssertion {
       return GridLayoutManagerColumnCountAssertion(expectedColumnCount)
     }
+
+    /**
+     * Verifies if an item at a specified position in a RecyclerView with the given ID contains a target view that is displayed.
+     *
+     * @param recyclerViewId The resource ID of the RecyclerView.
+     * @param itemPosition The position of the item in the RecyclerView.
+     * @param targetView The resource ID of the target view within the item.
+     */
+    fun verifyItemDisplayedOnListItem(
+      recyclerViewId: Int,
+      itemPosition: Int,
+      targetView: Int
+    ) {
+      onView(
+        atPositionOnView(
+          recyclerViewId = recyclerViewId,
+          position = itemPosition,
+          targetViewId = targetView
+        )
+      ).check(matches(isDisplayed()))
+    }
+
+    /**
+     * Verifies if an item at a specified position in a RecyclerView with the given ID does not contain a target view.
+     *
+     * @param recyclerViewId The resource ID of the RecyclerView.
+     * @param itemPosition The position of the item in the RecyclerView.
+     * @param targetView The resource ID of the target view within the item.
+     */
+    fun verifyItemDisplayedOnListItemDoesNotExist(
+      recyclerViewId: Int,
+      itemPosition: Int,
+      targetView: Int
+    ) {
+      onView(
+        atPositionOnView(
+          recyclerViewId = recyclerViewId,
+          position = itemPosition,
+          targetViewId = targetView
+        )
+      ).check(ViewAssertions.doesNotExist())
```

Comment:
Directly import ViewAssertions so that it is cleaner.
```suggestion
      ).check(doesNotExist())
```